charts/k8s-monitoring/values.yaml

```diff
@@ -292,37 +292,22 @@ metrics:
       # -- Does this port use TLS?
       isTLS: false
 
-    # -- The list of Kube State Metrics metrics that will be scraped by the Agent
-    # @default -- See [Allow List for Kube State Metrics](#allow-list-for-kube-state-metrics)
-    allowList:
-    - kube_daemonset.*
-    - kube_deployment_metadata_generation
-    - kube_deployment_spec_replicas
-    - kube_deployment_status_observed_generation
-    - kube_deployment_status_replicas_available
-    - kube_deployment_status_replicas_updated
-    - kube_horizontalpodautoscaler_spec_max_replicas
-    - kube_horizontalpodautoscaler_spec_min_replicas
-    - kube_horizontalpodautoscaler_status_current_replicas
-    - kube_horizontalpodautoscaler_status_desired_replicas
-    - kube_job.*
-    - kube_namespace_status_phase
-    - kube_node.*
-    - kube_persistentvolumeclaim_resource_requests_storage_bytes
-    - kube_pod_container_info
-    - kube_pod_container_resource_limits
-    - kube_pod_container_resource_requests
-    - kube_pod_container_status_last_terminated_reason
-    - kube_pod_container_status_restarts_total
-    - kube_pod_container_status_waiting_reason
-    - kube_pod_info
-    - kube_pod_owner
-    - kube_pod_start_time
-    - kube_pod_status_phase
-    - kube_pod_status_reason
-    - kube_replicaset.*
-    - kube_resourcequota
-    - kube_statefulset.*
+    cardinalityImprovements:
+      # -- Filter the list of metrics from Kube State Metrics to a useful, minimal set. See [Allow List for Kube State Metrics](#allow-list-for-kube-state-metrics)
+      useAllowList: true
+      # -- Metrics to be kept
+      includeMetrics: []
+      # -- Metrics to be dropped
+      excludeMetrics: []
```

Comment:
What's the default behaviour if these are set to `[]`? Might be worth a comment to clarify.

Reply:
Good call. I'll add some better comments in a future commit.